.circleci/config.yml

```diff
@@ -5,13 +5,13 @@ orbs:
 
 references:
   images:
-    ubuntu: &UBUNTU_IMAGE ubuntu-1604:202004-01
+    ubuntu: &UBUNTU_IMAGE ubuntu-2004:202201-02
 
 jobs:
   test:
     machine:
       image: *UBUNTU_IMAGE
-    shell: /usr/bin/env bash -euo pipefail -c
+    shell: /usr/bin/env bash -eo pipefail -c
```

Comment:
This was causing a `/etc/bash.bashrc: line 7: PS1: unbound variable` error, which resulted in `rvm` not setting the Ruby version correctly. See [the failing build](https://app.circleci.com/pipelines/github/hashicorp/vault-rails/137/workflows/0ca0311a-b007-4a97-8755-08cbc7c7352f/jobs/2908) for more details.

I'm pretty sure we were only setting this flag out of an abundance of caution, anyway, so no harm in un-setting it.